templates/hoover.nomad

```diff
@@ -104,104 +104,42 @@ job "hoover" {
-              {{- with service "hoover-es" }}
-                {{- with index . 0 }}
-                  location ~ ^/_es/(.*) {
-                    proxy_pass http://{{ .Address }}:{{ .Port }}/$1;
-                  }
-                {{- end }}
-              {{- end }}
-            {{- end }}
-
-            {{- range services }}
-              {{- if .Name | regexMatch "^snoop-" }}
-                {{- with service .Name }}
-                  {{- with index . 0 }}
-                    location ~ ^/{{ .Name | regexReplaceAll "^(snoop-)" "" }}/(.*) {
-                      proxy_pass http://{{ .Address }}:{{ .Port }}/$1;
-                      proxy_set_header Host {{ .Name | regexReplaceAll "^(snoop-)" "" }}.snoop.{{ key "liquid_domain" }};
-                    }
-                  {{- end }}
-                {{- end }}
-              {{- end }}
-            {{- end }}
-
-          }
-
-          {{- range services }}
-            {{- if .Name | regexMatch "^snoop-" }}
-              {{- with service .Name }}
-                {{- with index . 0 }}
-                  server {
-                    listen 80;
-                    server_name {{ .Name | regexReplaceAll "^(snoop-)" "" }}.snoop.{{ key "liquid_domain" }};
-                    location / {
-                      proxy_pass http://{{ .Address }}:{{ .Port }};
-                      proxy_set_header Host $host;
-                    }
-                  }
-                {{- end }}
-              {{- end }}
-            {{- end }}
-          {{- end }}
-
-          {{- if service "zipkin" }}
-            {{- with service "zipkin" }}
-              {{- with index . 0 }}
-                server {
-                  listen 80;
-                  server_name zipkin.{{ key "liquid_domain" }};
-                  location / {
-                    proxy_pass http://{{ .Address }}:{{ .Port }};
-                    proxy_set_header Host $host;
-                  }
-                }
-              {{- end }}
-            {{- end }}
-          {{- end }}
-          server_names_hash_bucket_size 128;
-          EOF
-        destination = "local/collections.conf"
-      }
-      config = {
-        image = "nginx:1.17"
-        port_map {
-          nginx = 80
-        }
+      config {
+        image = "fabiolb/fabio:1.5.11-go1.11.5"
         volumes = [
-          "local/collections.conf:/etc/nginx/conf.d/collections.conf:ro",
+          "local/fabio.properties:/etc/fabio/fabio.properties"
         ]
-        labels {
-          liquid_task = "hoover-collections-nginx"
+        port_map {
+          ui = 9991
+          lb = 9990
         }
       }
+      template {
+        destination = "local/fabio.properties"
+        data = <<-EOH
+        registry.backend = consul
+        registry.consul.addr = ${consul_url}
+        registry.consul.checksRequired = all
+        registry.consul.tagprefix = snoop-
+        ui.addr = :9991
+        ui.color = blue
+        proxy.addr = :9990
+        EOH
```

Comment:
You can indent the whole block as long as you indent `EOH` too and don't have empty lines (or the empty lines are also indented).